.storybook/preview.tsx

```diff
@@ -14,8 +13,11 @@ Object.defineProperty(NextImage, "default", {
 });
 
 addDecorator((story) => {
-  globalCss();
+  globalStyles();
   return <Box css={{ padding: "$4", backgroundColor: "$white" }}>{story()}</Box>;
 });
 
 configure([require.context("../src/", true, /\.stories\.(tsx|mdx)$/)], module);
+function globalStyles() {
```

Comment:
this will crash stuff in storybook. Remove this and import `src/theme/global-style.ts`